### PR TITLE
Add flashloans

### PR DESCRIPTION
**Pull request by Jam516**
Brief comments on the purpose of your changes:

Adding labels for wallets that have used flashloans on Aave, Uniswap and Balancer.

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Jam516

**Comment by jeff-dude** [comment]
please revisit why the accepted values test from `labels_infrastrucutre.sql` is failing here

**Comment by Hosuke** [comment]
> please revisit why the accepted values test from `labels_infrastrucutre.sql` is failing here

I am trying to figure out this problem.

**Comment by Hosuke** [comment]
Hi @jeff-dude, do you think this may be a transient server problem?

All records with `category` field value `infrastructure` in `test_schema.git_e69cb4f4_labels_flashloans_ethereum`:
https://dune.com/queries/2355450

While there are still records with `category` field values other than `infrastructure` in `test_schema.git_e69cb4f4_labels_infrastructure`:
https://dune.com/queries/2355468

File: models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                'flashloan_users'           AS model_name,
+                'ethereum'                  AS blockchain,
+                'Uniswap v3 Flashloan User' AS name
+FROM
+    {{ source('uniswap_v3_ethereum', 'Pair_evt_Flash') }}
+WHERE
+    amount0 != 0
+)
+
+SELECT *
```
Comment (jeff-dude) [inline models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql:59]: @Hosuke it looks like it's column order in the select here is incorrect, which is then union'ed into the `labels_infrastructure` spell. in your [query](https://dune.com/queries/2355468), you can see values aren't in the right category. please ensure this spell follows this pattern:
- blockchain
- address
- name
- category
- contributor
- source
- created_at
- updated_at
- model_name
- label_type

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hosuke) [reply models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql:59]: I see, using `select *` in roll-up spell is a dangerous practice.
I will try to solve all ambiguous column matching issues under `labels_infrastructure` in this PR.

[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/labels/addresses/infrastructure/persona/flashloans/labels_flashloans_ethereum.sql:59]: dangerous indeed :)

**Review by jeff-dude — APPROVED** [review]
thanks for using this to help clean up 🤝 and thanks for contributing @Jam516

**Pull request merged**